### PR TITLE
chapter 2 modifications

### PR DESCRIPTION
**Pull request by tomichec**
+ tip about TOR Browser when using blockchain
+ adds Electrum as popular lightning wallet

### CODE REVIEW

**Comment by ghost** [comment]
Can also add more LN wallets in the list.

https://blog.bitrefill.com/top-11-lightning-network-wallets-bitrefill-328b5465b1b4

File: 02_getting_started.asciidoc [hunk 1]
```diff
@@ -24,6 +24,8 @@ The below is an inexhaustive list (in alphanumerical order):
 ====
 Note that if using Lightning explorers, just like in existing block explorers,
 the privacy can be a concern because, if users are careless, the website may track their IP addresses and collect their behavior records (for example, the nodes users are interested in).
+To reclaim your privacy and stay anonymous online, it is recommended to use The Onion Router (TOR) Browser footnote:[TOR Project can help you to defend yourself against tracking and surveillance and circumvent censorship  (https://www.torproject.org/download/).]
```
Comment (renepickhardt) [inline 02_getting_started.asciidoc:27]: yes and no. It seems kind of offtopic in the context of our book to give recommendations to people how to behave while browsing the web.

**Review by renepickhardt — COMMENTED** [review]
concept ACK. 

Not sure about general browsing tips. @aantonop wdyt?

File: 02_getting_started.asciidoc (hunk 1 above)
Comment (aantonop) [reply 02_getting_started.asciidoc:27]: Agree with @renepickhardt

**Comment by aantonop** [comment]
I would merge this, but it mixes together two unrelated topic changes (TOR recommendation and Electrum wallet). 
Also, Electrum is misspelled in this PR. 

Thanks for your contribution, please try again!

**Pull request closed without merging**